.github/workflows/validate-floorplan-queries.yaml

```
@@ -7,6 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: fregante/setup-git-user@v2
```

Because

> there is significant risk in sourcing actions from third-party repositories on GitHub

per [GH docs](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)

I'd prefer to just inline/hardcode what the action does:
```suggestion
      - run: git config --global user.name "GitHub Actions"
      - run: git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

---

Fine with me. But let's use the user who did the latest commit, so we keep doing the same that the nebula release was doing.